cloudbuild.yaml

```diff
@@ -13,4 +14,24 @@ steps:
   - '--also-make'
   id: 'MVN_PACKAGE'
 
+# Tag and push the openjdk image to a staging repository for integration testing
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['tag', '${_IMAGE}', '${_STAGING_IMAGE}']
+  id: 'TAG_STAGING_IMG'
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['push', '${_STAGING_IMAGE}']
+  id: 'PUSH_STAGING_IMG'
+
+# Build a 'builder' image for running integration tests
+- name: 'gcr.io/cloud-builders/docker'
```

Comment:
yeah, I'm getting convinced that it is not necessary to run this step every time we want to build the image. 
What do you think?

Reply:
Agreed. Adding issue #155 to track this, and I'll add a followup PR right after this one to use the prebuilt image.